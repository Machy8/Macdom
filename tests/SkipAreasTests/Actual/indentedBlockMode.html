style
	h1 {
	color: #123;
	}

script
	var a = 123;
	a ++;
	alert(a);

code
	var a = 123;
	a ++;
	alert(a);
	div #wrapper
		div #inner
			Some text here

div #wrapper
	SKIP
		div
			div
				div

<div>
	<div>
		SKIP
			div
				h1 Some text inside
	</div>
</div>
