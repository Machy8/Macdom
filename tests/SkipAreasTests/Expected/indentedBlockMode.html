<style>
	h1 {
	color: #123;
	}
</style>
<script>
	var a = 123;
	a ++;
	alert(a);
</script>
<code>
	var a = 123;
	a ++;
	alert(a);
	div #wrapper
		div #inner
			Some text here
</code>
<div id="wrapper">
		div
			div
				div
</div>
<div>
	<div>
				div
					h1 Some text inside
	</div>
</div>
